admittance_controller/include/admittance_controller/admittance_rule_impl.hpp

```diff
@@ -220,9 +221,8 @@ controller_interface::return_type AdmittanceRule::reset()
     ik_tip_to_control_frame_tf_ = tf2_transform;
     control_frame_to_ik_tip_tf_ = tf2_transform.inverse();
   } catch (const tf2::TransformException & e) {
-    // TODO(destogl): Use RCLCPP_ERROR_THROTTLE
-    RCLCPP_ERROR(rclcpp::get_logger("AdmittanceRule"), "LookupTransform failed from '" +
-    control_frame_ + "' to '" + ik_tip_frame_ + "'.");
+    RCLCPP_ERROR_SKIPFIRST_THROTTLE(
```

Comment:
Cool, I didn't know about this SKIPFIRST option

Reply:
Me neither! I just did a short research to check how to avoid output during first X seconds.